testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -3947,6 +3947,56 @@ static void checkRlikeFails(SqlOperatorFixture f) {
     f1.checkNull("BIT_LENGTH(cast(null as binary))");
   }
 
+  @Test void testBitGetFunc() {
```

Comment:
Maybe we could reuse the test code for BIT_GET and GETBIT, as the only difference between them is the function name.

Reply:
Would you mean to reuse the test code by parameterized test?
By the way, currently `testGetBitFunc` only contains basic test, and `testBitGetFunc` contains the full test cases.

Reply:
Yes, in that way we could test all cases for both of them.